testtools/UART_interface/input.sh

```diff
@@ -5,4 +5,4 @@
 set -e
 
 # basic setup of input script for python
-echo -e "help\r\nset_az_iothub $IOTHUB_CONNECTION_STRING\r\n$SERIAL_TASK\r\n\r\nversion" | cat > input.txt
+echo -e "help\r\nset_az_iothub $IOTHUB_CONNECTION_STRING\r\n\r\n$SERIAL_TASK\r\n\r\nversion" | cat > input.txt
```

Comment:
>\n\r\n$ [](start = 58, length = 7)

Is this change needed?

Reply:
it is not, but I thought I'd give a little more process time for the chip after setting the hub string.